lib/index.js

```diff
@@ -60,11 +60,11 @@ function findPage(page, preferredPlatform, preferredLanguage) {
       } else if (targets.length > 0 && hasLang(targets, preferredLanguage)) {
         targetLanguage = preferredLanguage;
         targetPlatform = targets[0].platform;
-        console.log(`Command ${page} does not exist for the host platform. Displaying the page from ${targets[0].platform} platform`);
+        console.log(`Command ${page} does not exist for the host platform. Displaying the page from ${targetPlatform} platform`);
       } else if (targets.length > 0 && hasLang(targets, 'en')) {
         targetLanguage = 'en';
         targetPlatform = targets[0].platform;
-        console.log(`Command ${page} does not exist for the host platform. Displaying the page from ${targets[0].platform} platform`);
+        console.log(`Command ${page} does not exist for the host platform. Displaying the page from ${targetPlatorm} platform`);
```

Comment:
And I made yet another spelling mistake 🤦

Reply:
```suggestion
        console.log(`Command ${page} does not exist for the host platform. Displaying the page from ${targetPlatform} platform`);
```